maven-embedder/src/main/java/org/eclipse/sisu/plexus/PlexusXmlBeanConverter.java

```diff
@@ -43,6 +42,7 @@
 import org.codehaus.plexus.util.xml.pull.MXParser;
 import org.codehaus.plexus.util.xml.pull.XmlPullParser;
 import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.eclipse.sisu.Priority;
```

Comment:
out of curiosity why changing this package?

Reply:
Historically, Sisu relied on JSR250 and alike annotations, but as time passed, it implemented more and more of "own" annotations. Since a while (0.3.4 or so) it has it all in own namespace as well. If you look at history of this PR, you will see that I even removed `java.annotations:javax.annotations-api` from Maven completely, and Maven has happy about it, but we have one IT that uses it, the MavenITmng6084Jsr250PluginTest, hence I had to undo that change.

Given Sisu is "internal implementation detail" of Maven Core, I think using Sisu annotations _within_ Maven core is best/better, than relying on something else.

So this PR just makes sure that Maven Core uses Sisu annotations instead of something else, while the semantics 100% remains same as test/ITs prove.